app/profile/page.tsx

```
@@ -28,55 +28,59 @@ import {
+//     title: 'Echoes of Tomorrow',
+//     excerpt: 'The quantum resonance chamber hummed with possibility...',
+//     coverImage: '/stories/echoes-cover.jpg',
+//     date: '1 week ago',
+//     genre: 'Science Fiction',
+//     likes: 567,
+//     comments: 45,
+//     isNFT: true,
+//   },
+//   // ... more stories
+// ];
+const [profile, setProfile] = useState<any>(null);
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)